src/plugins/zoom.ts

```diff
@@ -66,6 +93,15 @@ class ZoomPlugin extends BasePlugin<ZoomPluginEvents, ZoomPluginOptions> {
     }
     this.container = this.wrapper.parentElement as HTMLElement
     this.wrapper.addEventListener('wheel', this.onWheel)
+
+    // Prevent native scrolling obstructing the zoom on this waveform
+    document.addEventListener('wheel', this.preventScroll, { passive: false });
```

Comment:
Just noticed this is assigned to the entire document element. Will it essentially prevent the page from scrolling whenever wavesurfer with Zoom is on the page? Can it be scoped to only the wavesurfer container?

Reply:
Yes your right it should be attached to a different container. Otherwise, as you mentioned, it prevents scrolling for the entire page. I'll take another look at this.